Content.Shared/Doors/Systems/SharedDoorSystem.cs

```diff
@@ -445,7 +445,9 @@ public IEnumerable<EntityUid> GetColliding(EntityUid uid, PhysicsComponent? phys
             if (otherPhysics == physics)
                 continue;
 
-            if (!otherPhysics.CanCollide)
+            //TODO: Make only shutters ignore these objects upon colliding instead of all airlocks
+            // Excludes Glasslayer for windows, GlassAirlockLayer for windoors, TableLayer for tables
+            if (!otherPhysics.CanCollide || otherPhysics.CollisionLayer == (int) CollisionGroup.GlassLayer || otherPhysics.CollisionLayer == (int) CollisionGroup.GlassAirlockLayer || otherPhysics.CollisionLayer == (int) CollisionGroup.TableLayer)
```

Comment:
Use a mask...

Reply:
Do you mean using TableMask and TableTopMachineMask instead of the layers, or do you mean I should create a new ShutterMask?